rewrite microsoft teams webhook (#36397)
demisto/content: Packs/MicrosoftTeams/Integrations/MicrosoftTeamsWebhook/README.md

| team_webhook | The alternative webhook for a different team.  If not defined, the integration's default webhook is used. | Optional | 
| alternative_url | The alternative URL to send in place of the link to the XSOAR Investigation. | Optional | 
| url_title | The title for the link, defaults to "Cortex XSOAR URL". Default is Cortex XSOAR URL. | Optional | 

> | adaptive_cards_format | When using webhook -Should the adaptive cards format be used?, When using Workflow- Should be a card or a single text message. | Optional | 

Reviewer: ```suggestion
| adaptive_cards_format | When using webhook -Should the adaptive cards format be used? When using Workflow- Should use a card or a single text message. | Optional | 
```

#### Context Output

There is no context output for this command.

## Troubleshooting
By default the message is being sent with a message template: `X Used a Workflow template to send this card`.

> In order to eliminate this line you can follow the following approach:

Reviewer: ```suggestion
In order to eliminate this line you can use the following approach:
```

> - Navigate to Microsoft’s Power Automate portal and sign into your Microsoft Teams account where you’ve previously [set up](https://make.powerautomate.com/) the Flow.
> - Click on “My flows” from the left side menu.
> - Click on the newly created Flow to open its details page.
> - On the Flow’s details page, click on “Save As” button on top.
> - Give your new Flow a name and click on the “Save” button.
> - Navigate back to “My flows” from the left side menu.
> - Find the copy Flow and click its name to access its details page.
> - On the Flow’s details page, click on the “Turn On” button at the top.
> - In order to find the new url link navigate to the edit tab on the Flow’s details page.
> - Click on the action task and copy the HTTP URL.
> - Configure an instance of the integration and add the copied Workflow URL for the Teams channel.

Reviewer: ```suggestion
1. Navigate to Microsoft’s Power Automate portal and sign into your Microsoft Teams account where you’ve previously [set up](https://make.powerautomate.com/) the Flow.
2. Click **My flows** from the left side menu.
3. Click the newly created Flow to open its details page.
4. On the Flow’s details page, click **Save As**.
5. Give your new Flow a name and click **Save"".
6. Navigate back to **My flows** from the left side menu.
8. Find the copy Flow and click its name to access its details page.
9. On the Flow’s details page, click **Turn On**.
10. In order to find the new URL link, navigate to the **Edit** tab on the Flow’s details page.
11. Click the action task and copy the HTTP URL.
12. Configure an instance of the integration and add the copied Workflow URL for the Teams channel.
```
